test/xref2/with.t/run.t

```diff
@@ -0,0 +1,48 @@
+This test checks that when we have expressions of the form `sig ... end with ...` that we replace
+them with simple `sig ... end` - any types or modules referred to in the `with` clauses will not
+be able to link anywhere because we don't render the inner signature anywhere.
+
+  $ cat test.ml
+  module type S = sig
+    type t
+    type u
+  end with type t = int with type u = int
+  
+
+  $ cat compile.sh
+  #!/bin/bash
+  
+  ocamlc -c -bin-annot test.ml
+  odoc compile page.mld --child module-test
+  odoc compile -I . --parent page test.cmt
+  odoc link -I . test.odoc
+  odoc html-generate --indent -o html test.odocl
+  odoc support-files -o html
+  
+
+  $ ./compile.sh
```

Comment:
```suggestion
  $ bash ./compile.sh
```
Absolute paths to `bash` can't work in general, a less wrong line would be `#!/usr/bin/env bash`, but it still won't work in some distros. It's generally better to not use them.

Reply:
It doesn't require bash, I'll change the #! line to /bin/sh - that should be sufficient.